test-configs.yaml

```diff
@@ -1901,6 +1943,11 @@ test_configs:
       - usb
       - v4l2-compliance-uvc
 
+  - device_type: fsl-s32v234sbc
```

Comment:
This one doesn't seem to be getting any jobs from staging.kernelci.org:
https://lavalab.nxp.com/scheduler/device_type/fsl-s32v234sbc
so it will take some investigation to find out why that is not happening.

Reply:
Hi,
I did not see any jobs coming from kernel-ci.
Please let me know if you need any info on the same.

Reply:
The problem is that there's no upstream DTB named `fsl-s32v234sbc`.  
@suram-nxp Is this board supported upstream?  If so what is the upstream DTB name?  Ideally we want the LAVA device-type name to match the upstream DTB name.

Reply:
If the issue is the dtb name not matching the LAVA device name, this can be resolved using the `dtb` attribute to specify the name to use (for example, see the `arndale` device type).

If support for this device type is not available upstream, then I would suggest to remove it from this PR so the others can be added now and create a new PR for it when it's ready.

Reply:
There is issue on my side.  I will remove s32 for now and update the PR.  I am sorry for the confusion.